lib/app.jsx

```diff
@@ -473,7 +503,10 @@ export const App = connect( mapStateToProps, mapDispatchToProps )( React.createC
 					</ContextMenu>
 				}
 				{ state.authorized ?
-						<div className={mainClasses}>
+						<div className={mainClasses}
+							onMouseMove={ makeResizable( { isResizing, sourceListSelector: '.source-list', onResizable: this.props.setSourceListCanResize, onUpdateWidth: this.props.setSourceListWidth } ) }
+							onMouseDown={ startResizing( { canResize, onStartResizing: this.props.setSourceListResizing } ) }
+							onMouseUp={ stopResizing( { isResizing, onStopResizing: () => this.props.setSourceListResizing( false ) } ) } >
```

Comment:
@beaucollins I'm all for removing component state, but I'm not sure `isResizing` belongs in the global app state either. How do you feel about the kind of wrapper component I provided above?

If the app is rendering the data contained in the state tree, then I'm not sure anything beyond the required data should be there. If we persisted the entire state and reloaded it when `isResizing = true` then wouldn't we end up in a wrong state?